tests/SbrpTests/Config.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+
+namespace SbrpTests;
+
+internal static class Config
+{
+    public const string RepoRootEnv = "SBRP_TESTS_REPO_ROOT";
+    public const string BuildTypeEnv = "SBRP_TESTS_BUILD_TYPE";
```

Comment:
Who sets those environment variables? We in runtime use runtimeconfiguration.json values instead of env vars: https://github.com/dotnet/runtime/blob/a9791e69991ad542d6bbf1d9a8e77127210499bb/src/tools/illink/test/Mono.Linker.Tests/Mono.Linker.Tests.csproj#L44-L46

Those are then read like this: https://github.com/dotnet/runtime/blob/a9791e69991ad542d6bbf1d9a8e77127210499bb/src/tools/illink/test/Trimming.Tests.Shared/PathUtilities.cs#L29C1-L30

Reply:
They can be set locally by the user or they can be set by the pipeline (https://github.com/dotnet/installer/blob/main/eng/pipelines/templates/jobs/sdk-diff-tests.yml#L101 as an example of a pipeline setting environment variables).